static-site/content/blog/2024-12-10-auspice-2024.md

```diff
@@ -7,104 +7,115 @@ sidebarTitle: "Notable changes in Auspice"
 
 The Nextstrain team continues to improve the visualisation capabilities of
 Auspice, including new features, bug fixes and performance improvements. This
-year we've released 18 versions and wanted to highlight some of the significant
-changes.
+year, we've released 19 versions and wanted to highlight some of the
+significant changes.
+
+<!--
+number of git tags in 2024 (generated by GitHub Copilot):
+git tag --list --format='%(creatordate:short)' | grep '^2024' | wc -l
+-->
+
+## Table of contents
+
+- [Focus on Selected](#focus-on-selected)
+- [View in Other Platforms](#view-in-other-platforms)
+- [Performance improvements](#performance-improvements)
+- [Conveying confidence of tips through saturation (version 2.56)](#conveying-confidence-of-tips-through-saturation-version-256)
+- [Filtering on all metadata](#filtering-on-all-metadata)
+- [Full changelog](#full-changelog)
```

Comment:
This would be nice but `id` tags aren't rendered by marked even though they [should be](https://github.com/markedjs/marked/blob/10020d018d4448b1d2d4612baf650c4e3a5949bf/docs/INDEX.md?plain=1#L13)?

Reply:
`id` is in our [allowed attributes](https://github.com/nextstrain/nextstrain.org/blob/b5af08c677e0053f342ee26f3eba968e6d0b9da1/static-site/app/blog/parseMarkdown.ts#L74), so it should work...

Based on the example you linked to, it seems like it doesn't add id tags automatically and you would have to change the Markdown titles to html

```md
<h2 id="focus-on-selected"> Focus on Selected </h2>
```

Reply:
Oh right 🤦 some reason I thought that was the rendered version but obviously not.

Looks like it's possible to [extend](https://github.com/markedjs/marked/blob/10020d018d4448b1d2d4612baf650c4e3a5949bf/docs/USING_PRO.md#the-renderer--renderer) the renderer to add an anchor automatically, I'll take a crack at it. If it doesn't work out, I'll use HTML tags instead.

Reply:
Created a separate PR for this: #1091